src/manifest.json

```diff
@@ -166,6 +164,13 @@
       "matches": ["*://*/*"],
       "run_at": "document_start",
       "js": ["js/contentScriptOnStart.bundle.js"]
+    },
+    {
+      "allFrames": true,
+      "matches": ["file://*/*", "http://*/*", "https://*/*"],
+      "runAt": "document_start",
+      "js": ["js/inpageScript.bundle.js"],
+      "world": "MAIN"
```

Comment:
I couldn't find any docs about this "world" property. Is it needed?

Reply:
this is the key for the solution. this will execute the script in browser envrionment rather than in isolate environment. new changes introduce world property which hold 2 values
1. isolated
2. MAIN

by default content scripts run in isolated environment